packages/cli/src/lib/environment/configEnvironment.js

```diff
@@ -30,6 +34,25 @@ function getEnv(arapp, environment) {
   }
 }
 
+const configureAPM = (ipfsRPC, gateway, ensRegistryAddress) => {
```

Comment:
Please add JSDocs to this function to known the data structures of `ipfsRPC, gateway, ensRegistryAddress`. If not full at least parcial :pray:

Reply:
Let's do it after release

Reply:
Sounds good